semgrep-core/src/utils/Unit_pcre_settings.ml

```diff
@@ -4,15 +4,15 @@
 
 let test_match_limit_ok () =
   let rex = SPcre.regexp "(a+)+$" in
-  match SPcre.pmatch ~rex "aaaaaaaaaaaaaaaaaaa!" with
+  match SPcre.pmatch ~rex "aaaaaaaaaaaaaaaaa!" with
```

Comment:
you needed to change the test to pass the limit? the longer aaa would exhaust  the 1M limit we set?

Reply:
Yes indeed. I'm hoping this only affects synthetic cases like this or to extreme cases like with `semgrep_targets.txt`.